hack/local-up-cluster.sh

```diff
@@ -401,6 +401,14 @@ contexts:
     name: local-up-cluster
 current-context: local-up-cluster
 EOF
+
+    # flatten the kubeconfig files to make them self contained
```

Comment:
> no need for the temporary file. Just pipe the HERE document above through kubectl config view --kubeconfig /dev/stdin --minify --flatten

I tried here: https://gist.github.com/deads2k/18d7d352671fbb50d2c362a326eae489, but the kubeconfigs were always empty.